tests/__wscache__/index.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+const path = require('path');
+
+// collect all json files
+/* 
+{ [methodName] => {
+    [key] => json
+} }
+ */
+export const WS_CACHE = {};
+
+const cacheFiles = dir => {
+    const dirFiles = fs.readdirSync(dir);
+    dirFiles.forEach(file => {
+        const filePath = path.resolve(dir, file);
+        if (fs.statSync(filePath).isDirectory()) {
+            cacheFiles(filePath);
+        } else if (file.endsWith('.json')) {
+            const method = path.basename(dir);
+            if (!WS_CACHE[method]) WS_CACHE[method] = {};
+            const key = file.replace('.json', '');
+            try {
+                const rawJson = fs.readFileSync(filePath);
+                const content = JSON.parse(rawJson);
+                WS_CACHE[method][key] = content;
+            } catch (error) {
+                console.error(`WS_CACHE parsing error: ${filePath}`);
+                throw error;
+            }
+        }
+    });
+};
+
+// read cache directory
+cacheFiles(path.resolve(__dirname));
```

Comment:
We export an empty global variable that is later populated. I'd much prefer if the `cacheFiles` function *returned* the results instead of mutating global var (Which is `const`!) and we'd just
```js
export const WS_CACHE = cacheFiles(path.resolve(__dirname));
```

What do you think?

Reply:
doesn't really matter since this script will execute before tests, but i've changed it for both 46d2f03 and d3599f8

Reply:
Thanks!